staging/automation/jleechanorg_pr_automation/tests/test_fixpr_prompt.py

```
@@ -13,7 +13,7 @@ class _FakeDispatcher:
     def __init__(self) -> None:
         self.task_description = None
 
-    def analyze_task_and_create_agents(self, task_description: str, forced_cli: str = "claude"):
+    def analyze_task_and_create_agents(self, task_description: str, forced_cli: str = "claude", wrap_prompt: bool = False, **kwargs):
```

Ruff (0.15.1)

[warning] Missing return type annotation for private function `analyze_task_and_create_agents`

(ANN202)